architecture/core_concepts/routes.adoc

```diff
@@ -621,8 +644,38 @@ the destination. This field is required, but only for re-encryption.
 [[router-sharding]]
 == Router Sharding
 
+Sharding allows the operator to partition the routes (via namespace/selectors)
+to be served by different router instances. And we could easily partition the
+routes in such a manner that the routers service overlapping sets (ala
+namespaces [A-J] *, [K-T] * and the last one is overlapping with the previous
+[Q-Z] *). Traditional sharding/distribution in such as case becomes a special
+case of partitioning with no overlapping sets.
+
+When routers are sharded, a given route is bound to a router in the group. The route
```

Comment:
A route can be served by multiple routers... it depends what label selectors the router is configured with.  This paragraph seems to be saying a route is served by at most one router.  Whereas, a route can be served by zero or more routers.